website/static/js/pages/gettingstarted-page.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// Reusable function to fix affix widths to columns.
+function fixAffixWidth() {
+    $('.affix, .affix-top, .affix-bottom').each(function (){
+        var el = $(this);
+        var colsize = el.parent('.affix-parent').width();
+        el.outerWidth(colsize);
+    });
+}
+
+$(document).ready(function () {
+
+    $(window).resize(function (){ fixAffixWidth(); });
+    $('.scrollspy').on('affixed.bs.affix', function(){ fixAffixWidth(); });
+
+  });
```

Comment:
It looks like you missed this section, you can remove this code and add only the instantiation like others.